test/router-luatest/router_2_2_test.lua

```diff
@@ -1222,9 +1224,10 @@ g.test_replicaset_services_do_not_spam_same_errors = function(g)
         end
     end)
     local msg = "Ping error from .*replica_2_a"
-    g.router:wait_log_exactly_once(msg, {timeout = 0.1, on_yield = function()
-        _G.failover_wakeup()
-    end})
+    t.helpers.retrying({}, function()
```

Comment:
We cannot retry here, the whole sense of `wait_log_exactly_once` is that logs are not duplicated. And if it failed once, it'll fail on all iterations, since the whole log after restart of the instance is checked